app-backend/db/src/main/resources/migrations/V74__add_session_to_label.sql

```diff
@@ -0,0 +1,3 @@
+ALTER TABLE public.annotation_labels
+ADD COLUMN is_active BOOLEAN NOT NULL DEFAULT TRUE,
```

Comment:
I thought adding a column with default value would be a costly migration, but it turned out to be fine after postgres version 11: https://www.depesz.com/2018/04/04/waiting-for-postgresql-11-fast-alter-table-add-column-with-a-non-null-default/